src/lib/svd.ml

```diff
@@ -0,0 +1,11 @@
+
+open Lacaml.D
```

Comment:
Yes, I'm going to bite the bullet. There's a bit more work to finish this pull request.